web/templates/channel.html

```diff
@@ -6,7 +6,7 @@
 <body>
   <div id="channel_view" class='channel-view'></div>
 <script>
-window.setup_channel_page({{ .Props }}, {{ .Team }}, {{ .Channel }}, {{ .User }});
+  window.setup_channel_page({{ .Props }}, {{ .Team }}, {{ .Channel }}, {{ .Preferences }});
```

Comment:
I thought removing `{{.User}}` was a mistake but it looks like it wasn't being used before anyways, despite being passed in